crates/dojo-core/src/world.cairo

```diff
@@ -135,4 +212,23 @@ mod World {
     fn set_executor(contract_address: ContractAddress) {
         executor::write(contract_address);
     }
+
+    // Internals
+
+    // Assert that calling system has authorization to write to the component
+    fn assert_auth(system: ClassHash, component: ClassHash) {
```

Comment:
Can we use `is_authorized` rather than having this method?

Reply:
That's true, it's a bit redundant having these two methods